### ISSUE TITLE
Hook for a matplotlib function

### ISSUE THREAD

**Issue by lsdR94** [pyviz/holoviews#3732]
While using the bokeh backend to plot a Curve element, I would like to use the fill_between function (matplotlib.pyplot.fill_between) that matplotlib has to color the area between two curves. 

I thought I could make it possible by using hooks, but I don't know the proper plot.handle to do that. The following picture illustrates what I meant.

![image](https://user-images.githubusercontent.com/45667851/58362930-1b87dd00-7e6b-11e9-825b-6a0e2a086d58.png)

**Comment by poplarShift** [comment]
You cannot use natively use matplotlib functions in the bokeh backend, the two are entirely different worlds. You have two choices: 
1. Either use the matplotlib backend, and use a hook as described at http://holoviews.org/user_guide/Customizing_Plots.html#Plot-hooks
2. You should also be able to achieve the same without hooks, using http://holoviews.org/reference/elements/bokeh/Area.html

Also for the future, consider posting usage questions like these on stackoverflow :)

**Comment by jbednar** [comment]
Right, (a) SO is better for such questions, and (b) you should be able to easily get a plot like that by overlaying a thresholded version of your data plotted as hv.Area on top of one plotted as hv.Curve.
